app/src/main/java/com/edulectronics/tinycircuit/Controllers/CircuitController.java

```diff
@@ -121,6 +121,16 @@ public int run(Activity circuitActivity) {
         return animator == null ? 0 : animator.delay;
     }
 
+    public Graph getGraph() {
```

Comment:
Deze methode kan ook gebruikt worden in de run() methode. Dit voorkomt dat er op twee plekken dezelfde code staat